q2_diversity/_alpha/_visualizer.py

```diff
@@ -167,6 +167,11 @@ def alpha_correlation(output_dir: str,
             "visualizer requires at least one numeric metadata column to "
             "execute.")
 
+    # save out metadata for download in viz
```

Comment:
Can we take advantage of the Metadata API for this? See `longitudinal volatility` for an example of [merging](https://github.com/qiime2/q2-longitudinal/blob/master/q2_longitudinal/_longitudinal.py#L189-L193) and [saving](https://github.com/qiime2/q2-longitudinal/blob/master/q2_longitudinal/_longitudinal.py#L244).